Makefile

```diff
@@ -1,5 +1,5 @@
 PACKAGE_NAME?=github.com/projectcalico/node
-GO_BUILD_VER?=v0.45
+GO_BUILD_VER?=v0.40
```

Comment:
I don't think we should revert the makefile version. Why did you need to do this?

Reply:
Well, this is just my cook to make it work. Thanks to point out.